docs/md/BasicTutorial.md

```diff
@@ -0,0 +1,282 @@
+# Tutorial
+
+
+
+## Installation
+Install the python client and check the installation as follows:
+
+```command
+$ pip install osparc
+$ python -c "import osparc; print(osparc.__version__)"
+```
+
+
+## Setup
+
+To setup the client, we need to provide a username and password to the configuration. These can be obtained in the UI under [Preferences > API Settings > API Keys](http://docs.osparc.io/#/docs/platform_introduction/main_window_and_navigation/user_setup/profile?id=preferences). Use the *API key* as username and the *API secret* as password. For security reasons, you should not write these values in your script but instead set them up via environment variables or read them from a separate file. In this example, we use environment variables
+
+```python
+
+import osparc
+
+cfg = osparc.Configuration(
+    username=os.environ["OSPARC_API_KEY"],
+    password=os.environ["OSPARC_API_SECRET"],
+)
+print(cfg.host)
+
+```
+
+The configuration can now be used to create an instance of the API client. The API client is responsible of the communication with the osparc platform
+
+
+The functions in the [osparc API] are grouped into sections such as *meta*, *users*, *files* or *solvers*. Each section address a different resource of the platform.
+
+
+
+For example, the *users* section includes functions about the user (i.e. you) and can be accessed initializing a ``UsersApi``:
+
+```python
+import osparc
+from osparc.api import UsersApi
+
+with osparc.ApiClient(cfg) as api_client:
+
+    users_api = UsersApi(api_client)
+
+    profile = users_api.get_my_profile()
+    print(profile)
+
+    #
+    #  {'first_name': 'foo',
+    #  'gravatar_id': 'aa33fssec77ea434c2ea4fb92d0fd379e',
+    #  'groups': {'all': {'description': 'all users',
+    #                     'gid': '1',
+    #                     'label': 'Everyone'},
+    #             'me': {'description': 'primary group',
+    #                    'gid': '2',
+    #                    'label': 'foo'},
+    #             'organizations': []},
+    #  'last_name': '',
+    #  'login': 'foo@itis.swiss',
+    #  'role': 'USER'}
+    #
+```
+
+
+## Solvers Workflow
+
+The osparc API can be used to execute any computational service published in the platform. This means that any computational service listed in the UI under the [Discover Tab](http://docs.osparc.io/#/docs/platform_introduction/core_elements/Discover?id=discover-tab) is accessible from the API. Note that computational services are denoted as *solvers* in the API for convenience, but they refer to the same concept.
+
+Let's use the sleepers computational service to illustrate a typical workflow.
```

Comment:
Could we add "The sleepers computational service takes as input one natural number and an optional text file input that contains another natural number, and then sleeps for the designated number of seconds based on inputs. It also provides two outputs: one natural number and a file containing a single natural number."